tests/unit/tasks/test_base_multi_text_input_task.py

```diff
@@ -0,0 +1,58 @@
+from tests.mocks.mock_prompt_driver import MockPromptDriver
+from griptape.structures import Pipeline
+from griptape.artifacts import TextArtifact
+from griptape.rules import Ruleset, Rule
+from tests.mocks.mock_multi_text_input_task import MockMultiTextInputTask
+
+
+class TestBaseMultiTextInputTask:
+    def test_string_input(self):
+        assert MockMultiTextInputTask(("foobar", "bazbar")).input[0].value == "foobar"
+        assert MockMultiTextInputTask(("foobar", "bazbar")).input[1].value == "bazbar"
+
+        task = MockMultiTextInputTask()
+        task.input = ("foobar", "bazbar")
+        assert task.input[0].value == "foobar"
+        assert task.input[1].value == "bazbar"
```

Comment:
It's good practice to check the length of the container (in this case `task.input`) to make sure there are no extra elements. (Though yeah, this is a trivial test and obviously won't fail given the simple implementation)